packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -758,6 +767,40 @@ export default class Meeting extends StatelessWebexPlugin {
     return this.webex.internal.metrics.submitCallDiagnosticEvents(payload);
   }
 
+  /**
+   * sets the network status on meeting object
+   * @param {String} networkStatus
+   * @private
+   * @returns {undefined}
+   * @memberof Meeting
+   */
+  setNetworkStatus(networkStatus) {
+    if (networkStatus === NETWORK_STATUS.DISCONNECTED) {
+      this.networkStatus = networkStatus;
+      Trigger.trigger(
+        this,
+        {
+          file: 'meetings/index',
+          function: 'setNetworkStatus'
+        },
+        EVENT_TRIGGERS.MEETINGS_NETWORK_DISCONNECTED,
+      );
+    }
+    else if (networkStatus === NETWORK_STATUS.CONNECTED) {
+      if (this.networkStatus === NETWORK_STATUS.DISCONNECTED) {
+        this.networkStatus = networkStatus;
+        Trigger.trigger(
+          this,
+          {
+            file: 'meetings/index',
+            function: 'setNetworkStatus'
+          },
+          EVENT_TRIGGERS.MEETINGS_NETWORK_CONNECTED,
+        );
+      }
+    }
```

Comment:
We could make it here without the network status actually being set, we should move 779 and 791 here `this.networkStatus = networkStatus;`